views/table.html

```diff
@@ -3,7 +3,12 @@
   <h3 class="s-q-item__title">{{ item.title }}</h3>{% endif %} {%- if item.subtitle and item.subtitle !== '' %}
   <div class="s-q-item__subtitle">{{ item.subtitle }}</div>
   {%- endif %}
-  <div style="overflow-x: auto">
+  <div style="overflow-x: auto;">
+    {%- if noInteraction !== true and item.options.showTableSearch === true %}
+    <div class="search-form" style="padding: 0 1px">
```

Comment:
why class and style?

Reply:
Lazyness. Will fix and also change the naming